src/main.ts

```diff
@@ -930,4 +936,17 @@ export async function reset(): Promise<void> {
   targetPlatforms = {}
 }
 
+function getNumberInput(name: string): number {
+  return parseInt(core.getInput(name))
+}
+
+export function getNumberArray(backoffIntervals: string): number[] {
+  const arrNum = String(backoffIntervals)
+    .split("")
```

Comment:
This kind of looks wrong, but we should be able to see it better if you use the github_action from your pull request branch. I'll sync up 1:1